ACCESSIBILITY.md

```diff
@@ -0,0 +1,90 @@
+
+### 3.0 How we work with Access-related Feedback
+
+Gathering access-related feedback is integral for ensuring that we can remain accessible as a community. We would like to offer a a few ways of sharing feedback with us, and are currently working to develop the following initiatives:
+
+1. **Feedback form**: Please submit access related comments to our [accessibility feedback form](https://forms.gle/Ngr2eUtQmf7aEeao6). All of the information in this form is processed anonymously, with the option of submitting contact information by choice. 
+2. **Drop-in sessions**: Throughout the calendar year, we would like to facilitate drop-in session to receive feedback about access related different parts of project. These may take place at our [Collaboration Cafe](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html) or may be hosted separately. 
+3. **Email the Research Community Manager or project team directly directly**: You can always email the Research Community Manager directly at asteele@turing.ac.uk with direct feedback. Alternatively, you can also reach out to her on Slack, or to any of the [project delivery team members](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md).
+4. **Attend the Accessibility Working Group meeting**: The Accessibility Working Group meetings may be a place where you can deliver feedback in a small group environment. These meetings are held on Zoom on the second Monday of each month at 17:00 UK time. Please join the #Accessibility channel on slack for more information.
+
+We recognise that many feedback forms and other forms for gathering feedback can often operate as "blackboxes" that receive information without taking actionable steps to address them.
+
+Knowing this, we are working to develop processes that enable us to action and report transparently on feedback given through these channels. This may include:
+- Reporting within the Accessibility Working Group in order to discuss possible actions that can be taken to address feedback. Currently, this process is under development.
+- Liasing with appropriate groups within _The Turing Way_ project (i.e. the Infrastructure Working Group) in order to create or resource technical changes that may be needed. Information about these working groups can be found on the [subpage of the Afterword](https://book.the-turing-way.org/afterword/subprojects)
+- Escalating feedback to the project co-leads, funders, and others at the constitutional level of the project as needed.
```

Comment:
```suggestion
Knowing this, we are working to develop processes that enable us to action and report transparently on feedback given through these channels. This may include:

- Reporting within the Accessibility Working Group in order to discuss possible actions that can be taken to address feedback.
  Currently, this process is under development.
- Liasing with appropriate groups within _The Turing Way_ project (for example, the Infrastructure Working Group) in order to create or resource technical changes that may be needed.
  Information about these working groups can be found in {ref}`subprojects`.
- Escalating feedback to the project co-leads, funders, and others at the constitutional level of the project as needed.
```

I think you would want _e.g._ instead of _i.e._, but I've changed it to "for example" to avoid Latin.

Reply:
Perfect, thanks for the internal citation as well @JimMadge